tests/adapters/switches/juniper_test.py

```diff
@@ -5352,6 +5394,7 @@ def compare(self, expected, actual):
                 else:
                     assert_that(actual[i].text is not None, "Node is " + node.tag)
                     assert_that(node.text.strip(), equal_to(actual[i].text.strip()))
+            assert_that(actual[i].attrib, has_length(len(node.attrib)))
```

Comment:
Maybe Move to [5390](https://github.com/internap/netman/pull/107/files#diff-034dde710188c9228ebdc20f875cf05bR5390) with his friends !